### PR TITLE
[luci/pass] Introduce QuantizeWeightsOnly visitor

### PR DESCRIPTION
**Pull request by glistening**
It introduces QuantizeWeightsOnly visitor.
It is subset of QuantizeWeights visitor.
It does not visit InstanceNorm, PReLU, ...

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

Related: https://github.com/Samsung/ONE/pull/11073, https://github.com/Samsung/ONE/issues/11057

### CODE REVIEW

**Review by jinevening — APPROVED** [review]
LGTM

File: compiler/luci/pass/src/QuantizeWeightsOnly.cpp
Comment (glistening) [inline compiler/luci/pass/src/QuantizeWeightsOnly.cpp:35]: - `iterate_per_channel` exists in `QuantizeWeights.cpp` and `QuantizeDequantizeWeightsPass`. Both are almost same, but slightly different. 

- It may be better to move this into `QuantizationUtils`. I don't know the history for the duplication (maybe intended). I guess several... performance? (by assuming channel_dim_index is always zero). But I am not sure.

Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWeightsOnly.cpp:35]: There is no performance reason. I think the function was simply missed during the past refactoring job. It would be great if you can help :)

**Comment by jinevening** [comment]
@glistening I'm waiting for a review that you requested to @hseok-oh.

**Comment by glistening** [comment]
@jinevening I added @hseok-oh to get 2 approvals. But @hseok-oh is exactly runtime guy like me. He may need time to read the code.

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**
